eslint.config.cjs

```diff
@@ -18,7 +18,7 @@ module.exports = [
       // External scripts
       ".pnp.cjs",
       ".pnp.loader.mjs",
-      "war/src/main/js/plugin-setup-wizard/bootstrap-detached.js",
+      "src/main/js/plugin-setup-wizard/bootstrap-detached.js",
```

Comment:
Part of the Webpack build invoked from the root Maven module, so the source file moves to the root Maven module.